packages/server/graphql/private/mutations/enableSAMLForDomain.ts

```diff
@@ -3,6 +3,7 @@ import {v4 as uuid} from 'uuid'
 import zlib from 'zlib'
 import getRethink from '../../../database/rethinkDriver'
 import {MutationResolvers} from '../resolverTypes'
+import {normalizeSlugName} from './helpers/SAMLHelpers'
```

Comment:
As I see, we currently use separate files for each helper, can we rename `SAMLHelpers.ts` file to `normalizeSlugName.ts` and do `export default`?